Packs/BitSight/Integrations/BitSightForSecurityPerformanceManagement/BitSightForSecurityPerformanceManagement.py

```diff
@@ -361,6 +366,9 @@ def get_company_findings_command(client, args, first_seen=None, fetch_incidents=
     res_json = client.get_company_findings(guid, first_seen, last_seen, severity_gte, grade, asset_category_eq,
                                            risk_vector)
 
+    return_results(res_json)
+    return
```

Comment:
what are you trying to achieve with those two lines? 
By doing so all the code below (lines 371-422) would be completely redundant.

Reply:
sorry, this 2 line was just to test the result. 
These must be removed

Reply:
alright, removed that for you